src/main/java/com/example/zoospringv1/AnimalDTO.java

```diff
@@ -0,0 +1,4 @@
+package com.example.zoospringv1;
+
+public record AnimalDTO (String species, String name) {
```

Comment:
Nawet jak w kursie będą nazywać AnimalDTO to olej ich, skrótowce piszemy tez CamelCase, takze prośba o AnimalDto :)